hack/xref-helpmsgs-manpages

```diff
@@ -639,6 +664,59 @@ sub podman_man {
                     $previous_format = $format;
                 }
             }
+            # Same as above, but with --filter
+            elsif ($previous_flag eq '--filter') {
+                if ($line =~ /^\|\s+(\S+)\s+\|/) {
+                    my $filter = $1;
+
+                    # Special-case transformations: some man pages use
+                    # asterisks, "*foo*", some don't. Ignore asterisks.
+                    $filter =~ tr/\*//d;
+                    # FIXME: There seems to be an invisible filter, "after",
+                    # documented in the man pages but not appearing in
+                    # completion. For podman-images, #13607 added "after"
+                    # to man pages. For podman-volume-*, it seems to be
+                    # there from the beginning.
+                    # QUESTION: do we want to accept "after" in completion,
+                    # or keep it hush-hush?
```

Comment:
Marking draft because I'd like an answer here: I think we should either complete `after`, or remove it from man pages.

Also draft because I have no idea what I'm doing, and my additions to `completion.go` need very thorough review.

Reply:
If it is a valid filter then we should complete it, my rule of thumb is if it is valid then we should complete it unless the option is hidden/deprecated.

Reply:
Thanks. Fixed.